ReadyPlayerOne/Controllers/PlayerController.cs

```diff
@@ -25,6 +25,8 @@ public IActionResult Add()
             ViewBag.Action = "Add";
             //Adding Player Images
             ViewBag.PlayerImages = context.PlayerImages.OrderBy(p => p.Image).ToList();
+            ViewBag.AlignmentTypes = context.Alignments.ToList();
```

Comment:
I am adding your lookups to the view bag in each of the locations you call Edit.cshtml